src/DucksNet.WebUI/Pages/Models/UpdateEmployeeModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace DucksNet.WebUI.Pages.Models;
+
+public class UpdateEmployeeModel
```

Comment:
Is the update tested?  My requests would fail no matter what.

Reply:
Yes, it is tested. The constraints must be updated (updated email/phone must not appear in the database)